packages/components/tests/integration/components/hds/form/character-count/index-test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module(
+  'Integration | Component | hds/form/character-count/index',
```

Comment:
at times we test also the `aria` attributes here; should we test that the `aria-live="polite"` is present in the component?

Reply:
good call – added in 9450771ad1db25abbd4c094edf45374d41492c62